nilrt-build-init.env

```diff
@@ -2,6 +2,21 @@
 
 SCRIPT_ROOT=$(realpath $(dirname ${BASH_SOURCE:-${0}}))
 
+enable_ni_org_conf=false
+positionals=()
+
+while [ $# -ge 1 ]; do case "$1" in
+	-o|--org)
+		enable_ni_org_conf=true
```

Comment:
I found the commit wording confusing and my bash knowledge is minimal, so let me confirm my understanding here. An NI user now has to invoke this script with the -o parameter `. nilrt-build-init.env -o` to copy the ni-org.conf file to the build directory. Correct?

Reply:
shruthi - yes that's my read, and I'm not completely happy with that solution.

@amstewart Could we possibly make the script a bit smarter and check the user's email address and if its an @ni.com email, initialize enable_ni_org_conf=true ?  'git config user.email |grep "\@ni.com$"'  - or some such ??  That way the script would basically do the right thing most of the time (assuming we all have (or set) our email addresses with git).

Reply:
I don't like the idea of the script trying to be smart like that. Having an NI email address configured doesn't _mean_ that you are connected to the corporate network. I disconnect from the VPN all the time and still want to do bitbake things, while my repo still has an NI email attached to it.

I've considered a few different heuristics for determining if you're connected to the corporate network. But none of them really get around the fundamental requirement that VPN builders understand that these config settings are a thing, and that they will only work while connected to the VPN.

I think its an improvement that they are broken out specifically from the rest of the local.conf. And I like the implication of the `-o` switch on the initscript; it implies "Yes, I understand the implications of adding these config settings and want to be responsible for turning them off again, if I need to."

As I said above, you can just wrap the script on your local machine with a one-line source file that provides the `-o` argument, or edit the initscript locally.

Reply:
Too true, you've got a couple good points there.  I guess I'll have to write a script or remember something ;)